### PR TITLE
Add -e flag for Extensions.

### PR DESCRIPTION
**Pull request by C0RB3N**
This adds an `-e` flag for extensions. This currently replaces %EXT% in a given wordlist, but we should soon add the option to append an extension to each line in the wordlist.

Example usage:

```bash
root@dev:~# ffuf -u http://example.com/FUZZ -e asp,html,jpg -w wordlist.txt

        /'___\  /'___\           /'___\
       /\ \__/ /\ \__/  __  __  /\ \__/
       \ \ ,__\\ \ ,__\/\ \/\ \ \ \ ,__\
        \ \ \_/ \ \ \_/\ \ \_\ \ \ \ \_/
         \ \_\   \ \_\  \ \____/  \ \_\
          \/_/    \/_/   \/___/    \/_/

       v0.9git
________________________________________________

 :: Method       : GET
 :: URL          : http://example.com/FUZZ
 :: Matcher      : Response status: 200,204,301,302,307,401,403
________________________________________________

:: Progress: [3/3] :: 0 req/sec :: Duration: [0:00:00] :: Errors: 0 ::
```

### CODE REVIEW

**Comment by joohoi** [comment]
Ah, now I finally understood about what you meant! I'll have to think about this a bit.

**Comment by codingo** [comment]
I pulled this and had a play with it. Works as intended, and personally find it to be quite a useful feature. Would be good to see it land.

**Comment by joohoi** [comment]
I described the current plan here: https://github.com/ffuf/ffuf/issues/19#issuecomment-480732407

TL;DR: Use this patch but hide the functionality behind a CLI flag for enabling DirSearch wordlist compatibility. Then add a ffuf-idiomatic extension appender functionality (all of the wordlist entries get `-e` extensions appended to them if defined by keyword).

**Comment by joohoi** [comment]
Closing this in favor of #25
